config/localTemplate.go

```diff
@@ -134,6 +134,8 @@ type Local struct {
 	// Estimating 1.5MB per incoming connection, 1.5MB*2400 = 3.6GB
 	IncomingConnectionsLimit int `version[0]:"-1" version[1]:"10000" version[17]:"800" version[27]:"2400"`
 
+	// P2PIncomingConnectionsLimit is used as IncomingConnectionsLimit for P2P connections in hybrid mode.
+	// For pure P2P nodes IncomingConnectionsLimit is used.
 	P2PIncomingConnectionsLimit int `version[34]:"1200"`
```

Comment:
While this is consistent, I think this could be confusing (along with similar settings) for node runners. Wondering if we should reach out and talk to folks @cce .

Reply:
I renamed both to P2PHybridXyz and these are used only in hybrid mode controlled by `EnableP2PHybridMode`. Now all naming is consistent.

There are two issues with the suggested approach of using both P2PIncomingConnectionsLimit and IncomingConnectionsLimit in P2P mode:
1. IncomingConnectionsLimit = 0 default value has a special meaning
2. The code becomes quite hairy with checking for both variants in multiple places.
One can argue (1) is not a big deal and I agree but it actually brings exact the same confusion.
(2) can be also avoided by refactoring `p2p.MakeHost` and `p2p.MakeService` to not accept `cfg config.Local` but instead functional params like p2p.Identity, p2p.NetAddress, p2p.IncomingConnectionsLimit so that this variation logic is concentrated in one place (p2pNetwork.go file).